### PR TITLE
My newdles frontend

### LINKED ISSUE #58: "My newdles" page
A page listing all the newdles the user has created and their current state (finished, ongoing, number of answers...).

### PR DESCRIPTION
**Pull request by mvidalgarcia**
closes #58 

image

### CODE REVIEW

File: newdle/client/src/components/MyNewdles.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      timezone: 'Europe/Zurich',
+    },
+  ];
+  return (
+    <Container text className={styles.newdles}>
+      {dummyData.map(newdle => (
+        <div
+          className={styles.newdle}
+          key={newdle.id}
+          onClick={() => console.log('go to newdle summary')}
```
Comment (ThiefMaster) [inline newdle/client/src/components/MyNewdles.js]: `const router = useRouter();` and then `router.history.push(`/newdle/${newdle.code}/summary`);`

Comment (meluru) [inline newdle/client/src/components/MyNewdles.js]: maybe let's leave this empty for now?

File: newdle/client/src/components/MyNewdles.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                    toMoment(newdle.finalDt).add(newdle.duration, 'm'),
+                    'HH:mm'
+                  )}`}{' '}
+                  ({newdle.timezone})
+                </label>
+              </span>
+            </div>
+          )}
+          <div className={styles.participants}>
+            <Icon name="users"></Icon>
```
Comment (ThiefMaster) [inline newdle/client/src/components/MyNewdles.js]: `<Icon name="users" />`

File: newdle/client/src/components/MyNewdles.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      {newdle.finalDt && (
+        <div className={styles.info}>
+          <span>
+            <Icon name="calendar alternate outline" />
+            <label>{serializeDate(newdle.finalDt, 'Do MMMM')}</label>
+          </span>
+          <span>
+            <Icon name="clock outline" />
+            <label>
+              {`${serializeDate(newdle.finalDt, 'HH:mm')} - ${serializeDate(
```
Comment (ThiefMaster) [inline newdle/client/src/components/MyNewdles.js]: Now that it's a separate content you could do the formatting before getting into the return jsx block, and thus avoid the awful wrapping here:

```
const startDate = ...;
const endDate = ...;
```

**Pull request merged**